code/game/objects/items/weapons/tanks/tank_types.dm

```diff
@@ -113,9 +113,12 @@
  */
 
 /obj/item/weapon/tank/internals/plasmaman
+	name = "plasmaman plasma tank"
+	desc = "A tank of plasma gas."
 	icon_state = "plasmaman_tank"
 	item_state = "plasmaman_tank"
 	force = 10

```

Comment:
That's way to much force? It should be the same force as a little tank not a big tank.